packages/language-server/src/lib/documents/configLoader.ts

```diff
@@ -152,6 +152,12 @@ export class ConfigLoader {
             let config = this.disabled
                 ? {}
                 : (await this.dynamicImport(pathToFileURL(configPath)))?.default;
+
+            if (!config) {
+                throw new Error(
+                    'Missing exports in the config. Make sure to include "export default config" or "module.exports = config"'
```

Comment:
@benmccann It's this error message ok to you?

Reply:
Yeah, that looks good and would help make it clearer what the issue is. Thanks!

Even more than that I'd really like `--ignore` to avoid trying to load the config in the first place, but this is still helpful

Reply:
One of the problems I can think of it's that the ignored file could still be imported by another file. And when the svelte-preprocess default language config is set in that errored config file. The ignored file might be recognized as the wrong script type(js or ts). Thus creating some confusing typing errors in like props type check. I'll have to think about how to suppress the error message more.